packages/react-ui/src/localization/id-ID.js

```diff
@@ -0,0 +1,77 @@
+const appMessages = {
```

Comment:
If this file is not required (just for demo purpose), I would recommend to delete it